src/ng2-datepicker/locales/ru-locale.ts

```diff
@@ -0,0 +1,23 @@
+import { ILocale } from "./locale-manager";
+
+export class RussianLocale implements ILocale{
+    firstWeekdaySunday(): boolean {
+        return false;
+    }
+    langIdentifier(): string {
+        return "ru";
+    }
+    daysOfWeek(): string[] {
+        return ["Вск", "Пн", "Вт", "Ср", "Чт", "Пт", "Сб"];
```

Comment:
no problem, tyt. and check you email btw, you should got invitation to collaborate.